paper-checkbox.js

```diff
@@ -85,12 +85,18 @@ const template = html`<style>
     pointer-events: none;
     -webkit-transition: background-color 140ms, border-color 140ms;
     transition: background-color 140ms, border-color 140ms;
+
+    transition-duration: var(--paper-checkbox-transition-duration, 140ms, 140ms);
+    -webkit-transition-duration: var(--paper-checkbox-transition-duration, 140ms, 140ms);
```

Comment:
I think the `-webkit-`-prefixed declarations should be first so that the normal ones override them if they're both supported. Also, there's an extra `, 140ms` in each of these.

Reply:
Order changed.
The additional 140ms is due to there's separate configuration for "background-color" and "border-color". 
I could remove one if you like.

Reply:
> I could remove one if you like.

That would be good; the `var()` syntax only accepts one or two parameters, i.e. `var(--x, y, z)` doesn't end up defaulting to `y, z` if `--x` isn't defined. But you don't need to worry about setting both in this case: if one transition declaration has fewer values than another, [the values are repeated to fill the needed values](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Transitions/Using_CSS_transitions#When_property_value_lists_are_of_different_lengths). So, `transition-duration: var(--paper-checkbox-transition-duration, 140ms);` will set `transition-duration` of both `background-color` and `border-color` to `140ms` if `--paper-checkbox-transition-duration` is not defined.

(If you did need to explicitly set both, you would need to do something like `prop: var(--x, y), var(--x, y);`, but the lists for `transition-` properties don't require this because of the repeating thing.)

Reply:
Actually I tried with case for setting them differently as 3 parameter in var, and it worked, at least in our internal polymer project. 
But of course I will remove the 3rd parameter.

Reply:
Oh! Sorry, you're right: https://developer.mozilla.org/en-US/docs/Web/CSS/var#Syntax . Apparently this is explicitly supported.